patches/minecraft/net/minecraft/block/BubbleColumnBlock.java.patch

```diff
@@ -0,0 +1,33 @@
+--- a/net/minecraft/block/BubbleColumnBlock.java
++++ b/net/minecraft/block/BubbleColumnBlock.java
+@@ -24,6 +24,7 @@
+ import net.minecraft.world.server.ServerWorld;
+ import net.minecraftforge.api.distmarker.Dist;
+ import net.minecraftforge.api.distmarker.OnlyIn;
++import net.minecraftforge.block.IBubbleSource;
+ 
+ public class BubbleColumnBlock extends Block implements IBucketPickupHandler {
+    public static final BooleanProperty field_203160_a = BlockStateProperties.field_208179_f;
+@@ -80,9 +81,11 @@
+       Block block = blockstate.func_177230_c();
+       if (block == Blocks.field_203203_C) {
+          return blockstate.func_177229_b(field_203160_a);
+-      } else {
+-         return block != Blocks.field_150425_aM;
+       }
++      if (block instanceof IBubbleSource) {
++         return ((IBubbleSource) block).isDragging();
```

Comment:
From my understanding of modifying vanilla code (that creates the patches) we are not supposed to add more imports so as to reduce the patch size. So that would make this become:
```java
if (block instanceof net.minecraftforge.block.IBubbleSource) {
    return ((net.minecraftforge.block.IBubbleSource) block).isDragging();
```
Realistically to further reduce the patch size it potentially should be (keeping the else that already exists):
```java
return !(block instanceof net.minecraftforge.block.IBubbleSource) || ((net.minecraftforge.block.IBubbleSource) block).isDragging();
```